.github/workflows/postman.yml

```diff
@@ -0,0 +1,70 @@
+          sed -i 's/ef71a9d6c63444fce6abd2df8fab2755cfb51f6794e578f60d99337193811842/"{{tx_id}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/hpyron_rr1qdukq6q534plnen70k95f80q45cktg0mhmzrzpfyz8jqmgqqqe/"{{token_rri}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/xrd_rr1qy5wfsfh/"{{native_token_rri}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/3046022100cda0fcd31bf976aa65c31c180b4f595a61c866252c52898b952b8fe1d8fdcc33022100999e83036f9529c6250c07423d8b57f252dfba1b843385970bfa69f2c367658e/"{{tx_bytes}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/0d00010776bf65acf2d25e9dcf4c716f5f39f201dccbfa173ad9c7f1c1dbcb8a86776b4d0000000101002100000000000000000000000000000000000000000000000000012901c1cf3900000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7ff8000800000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7f9402004506000402c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5010000000000000000000000000000000000000000000000000000000000000064000c0034303030303663366636313634323037343635373337343639366536373230373437323631366537333631363337343639366636650b00cda0fcd31bf976aa65c31c180b4f595a61c866252c52898b952b8fe1d8fdcc33999e83036f9529c6250c07423d8b57f252dfba1b843385970bfa69f2c367658e/"{{signed_transaction}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/0d00010776bf65acf2d25e9dcf4c716f5f39f201dccbfa173ad9c7f1c1dbcb8a86776b4d0000000101002100000000000000000000000000000000000000000000000000012901c1cf3900000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7ff8000800000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7f9402004506000402c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5010000000000000000000000000000000000000000000000000000000000000064000c003430303030366336663631363432303734363537333734363936653637323037343732363136653733363136333734363936663665/"{{unsigned_transaction}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798/"{{public_key_hex}}"/g' gateway-api-spec.yaml
+      - name: Update Postman's Gateway API collection from the spec
+        run: |
+          npx @apideck/portman -l gateway-api-spec.yaml --syncPostman true -o tmp.collection.json --envFile .env --cliOptionsFile cliopts.json \
+            --postmanUid 14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e
+      - name: Download the collection
+        run: |
+          curl -X GET -H "X-API-KEY:${{secrets.POSTMAN_API_TOKEN}}" https://api.getpostman.com/collections/14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e > tmp.collection.json
+      - name: Add a timestamp to the name
```

Comment:
Out of interest - why do we updated, then download, change a few things, and then re-update? I take it the timestamp and base url couldn't be fixed before hand?

Reply:
Yes. I have not found a way to change the baseUrl variable _name_ beforehad. The above the only way I can do it.

The timestamp one is similar. If you set the timestamp beforehand (in the `cliopts.json` file) then Postman will create an entirely new collection. So I have to download the collection, change it and then upload it.

Especially since they don't support a PATCH method :/